README.md

```diff
@@ -0,0 +1,15 @@
+travis.yml badges
+
+branch models-1
+[![Build Status](https://travis-ci.org/owy1/django-imager.svg?branch=master)](https://travis-ci.org/owy1/django-imager)
+
+branch front-end-1
+[![Build Status](https://travis-ci.org/owy1/django-imager.svg?branch=master)](https://travis-ci.org/owy1/django-imager)
+
+branch models-2
+[![Build Status](https://travis-ci.org/owy1/django-imager.svg?branch=master)](https://travis-ci.org/owy1/django-imager)
+
+branch front-end-2
+
+
```

Comment:
Your readme should include more useful things like how to set up the app from github, what routes your app has and what models are used.